packages/crypto-wallet-core/src/transactions/index.ts

```diff
@@ -22,6 +22,10 @@ export class TransactionsProxy {
   sign(params): string {
     return this.get(params).sign(params);
   }
+
+  encodeData(params) {
```

Comment:
where does this get used?

Reply:
https://github.com/bitpay/copay/pull/10162/files#diff-4803d1707f7b95d38465e857738498c1R792

encoding tx.data property when creating a tx to get correct gasLimit.